src/docs/Architecture.md

```diff
@@ -0,0 +1,32 @@
+## General Moments
```

Comment:
Why src/docs and not just docs? This is not source file, so let's put it into a separate directory in repository